ckanext/authz_service/actions.py

```diff
@@ -148,4 +148,4 @@ def _generate_jti(length=8):
     # type: (int) -> bytes
     """Generate a unique token ID
     """
-    return b''.join(random.choice(string.printable) for _ in range(length))
+    return ''.join(random.choice(string.printable) for _ in range(length))
```

Comment:
Is this not changing behavior on py2?

Reply:
This small changed is what made me feel insecure about the PR.

According to oficial docs, `b''` prefix [is ignored in Python 2](https://stackoverflow.com/questions/6269765/what-does-the-b-character-do-in-front-of-a-string-literal/6269785) and means that the literal should become a bytes literal in Python 3 which make sense because it is used as a payload.

Tests passed and I also test it manually to be confident that the method works and that the result can be used as payload. But string changes between 2 and 3 still confuses me. So if you have any deep knowledge about this it'd be appreciated.